homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -499,16 +462,16 @@ def device_state_attributes(self):
             'last_attempt': self.car.last_check,
             'updated_on': self.car.last_battery_response,
             'update_in_progress': self.car.request_in_progress,
-            'location_updated_on': self.car.last_location_response,
             'vin': self.car.leaf.vin,
         }
 
-    async def async_added_to_hass(self):
+    def added_to_hass(self):
         """Register callbacks."""
         self.log_registration()
-        async_dispatcher_connect(
+        dispatcher_connect(
             self.car.hass, SIGNAL_UPDATE_LEAF, self._update_callback)
 
+    @callback
     def _update_callback(self):
         """Update the state."""
         self.schedule_update_ha_state(True)
```

Comment:
Use `self.async_schedule_update_ha_state`.

Reply:
Fixed in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/ac58b4f55eb93c0f78d22a67ae69a625b2674e52